static/app/components/events/interfaces/performance/durationChart.tsx

```diff
@@ -132,8 +151,9 @@ function Content({allEvents, affectedEvents, loading, errored}) {
       grid={{left: '0', right: '0', top: '0', bottom: '0'}}
       height={200}
       series={affectedEvents}
+      seriesOptions={{smooth: true}}
```

Comment:
It's def looking better, but let's try "low" fidelity maybe and see if it looks a lot less spiky, since it should reduce some of the noise due to sparse data? Just give it a shot, I think the low fidelity intervals aren't great below 24h, but we can potentially fix them to have a reasonable number of buckets? 🤔 

Change (above this diff in the same file)
```ts
  const interval = getInterval({start, end: nowRef.current, utc: true}, "low");
```

Reply:
Low fidelity definetly made it look smoother!
<img width="457" alt="image" src="https://user-images.githubusercontent.com/44422760/184656385-509b65ce-e444-4ff3-b4df-9f2f23573455.png">

Reply:
I haven't tested <24h, could you clarify what you mean by "buckets"?

Reply:
Ah sorry, I just mean that given an interval for a timeperiod (eg. 6h interval for a 24h period) you have a certain number of buckets of data (eg. 4 buckets of 6h for a 24h period). How smooth / sparse data looks is directly related to picking the right # of buckets. Looks great thought!